src/index.html

```diff
@@ -1,17 +1,117 @@
 <!doctype html>
 <html lang="en">
+
   <head>
     <meta charset="UTF-8">
-    <meta
-      name="viewport"
-      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
-    >
+    <meta name="viewport"
+    content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>HTML Form</title>
     <link rel="stylesheet" href="./style.css">
   </head>
+
   <body>
-    <h1>HTML Form</h1>
     <script type="text/javascript" src="./main.js"></script>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="post">
+      <fieldset>
+        <legend>Personal information</legend>
+        <label class="selectors">
+        Surname:
+          <input 
+          type="text" 
+          name="Surname" 
+          autocomplete="off"
+          >
+        </label>
+        <label class="selectors">
+        Name:
+          <input type="text" name="Name" 
+          autocomplete="off">
+        </label>
+        <label class="selectors">
+        How old are You?
+          <input type="number" name="tentacles" 
+          min="1" max="100" 
+          value="12">
+        </label>
+        <label class="selectors">
+        Full date of birth:
+          <input type="date" name="trip-start">
+        </label>
```

Comment:
```suggestion
        <label class="selectors">
          Full date of birth:
          <input type="date" name="trip-start">
        </label>
```
this text and input are on the same level